composer.json

```diff
@@ -210,6 +210,7 @@
         "mikeyp/google_tag": "dev-8.x-1.x#3caa4f0b7f008576936ab7501d483bfcf58a4764",
         "phlak/semver": "^2.0",
         "phpunit/phpunit": "^6",
+        "symfony/dom-crawler": "3.4.35",
```

Comment:
Seems like we could be more loose here - `^3` would be fine, no?

Reply:
@weitzman Yeah, I don't think a test would have caught this particular issue, since the library existed on CircleCI, and is needed for the tests to run. I made the dom-crawler requirement looser. Should be fine. I was originally just matching what core-recommended's components use, but it's in step with the latest security version.